extensions/interactions/base_test.py

```diff
@@ -23,11 +23,11 @@
 
 from core.domain import dependency_registry
 from core.domain import exp_services
-from core.domain import html_validation_service
 from core.domain import interaction_registry
 from core.domain import obj_services
 from core.tests import test_utils
 from extensions.interactions import base
+from extensions.interactions.CodeRepl import CodeRepl
```

Comment:
Don't do this, use interaction_registry to retrieve the interaction class.

Reply:
Done